backend/migrations/20240406023149_intial_schema.sql

```diff
@@ -0,0 +1,217 @@
+CREATE TYPE user_role AS ENUM ('User', 'SuperUser');
```

Comment:
I don't think it's a good idea to group all migrations into a single file :skull: